### ISSUE TITLE
FRONT: autofocus for input/button

### ISSUE THREAD

**Issue by emmbryo** [emmbryo/quizzie#53]
## Autofocus for input fields and next button 
When playing a quiz game, the user should be placed in the input field if available, or on the next button. This to enable easier navigations with the keyboard only when playing a quiz.
